dt-core/admin/js/dt-utilities-workflows.js

```diff
@@ -1616,6 +1626,13 @@ jQuery(function ($) {
     response['html'] =
       `<textarea placeholder="Enter text..." style="min-width: 100%;" id="${window.dt_admin_shared.escape(response['id'])}">`;
 
+    let help_button = $('#workflows_design_section_step3_help_text');
+    const new_help_button = `<a id="workflows_design_section_step3_help_text">[<a class="dt-docs"
+        data-title="dt_right_docs_comments_title"
+        data-content="dt_right_docs_comments_content"
+        style="cursor: pointer;">&#63;</a>]</a>`;
+    help_button.replaceWith(new_help_button);
+
```

Comment:
You can remove the cursor style if you set an `href` attribute on the tag. That is a feature of all browsers that it will only change the cursor if it has an `href`. You can just set it to `href="#"` so it doesn't actually follow the link. The `#` is used to link to a specific HTML element by id on the page and will just go to the top if js doesn't prevent it, like we do with this.

Reply:
This help link never gets removed. If you select Comments from the dropdown and then select some other field, the help link is still there. Whenever the dropdown is changed, we should remove this help link. If we change it to a span, you can just remove all the content of the span. And then in this case when the Comments field is selected, you can append this link within the span.